src/routes/docs/tutorials/nextjs/step-2/+page.markdoc

```diff
@@ -0,0 +1,70 @@
+---
+layout: tutorial
+title: Create Next.js app
+description: Create a Next.js project using Appwrite.
+step: 2
+---
+
+# Create Next.js project {% #create-next-js-project %}
+
+Create a new Next.js project using the `create-next-app` command.
+
+```sh
+npx create-next-app expense-tracker --eslint
+```
+
+We are using the `--eslint` flags to add ESLint to our project.
+
+# Install dependencies {% #install-dependencies %}
+
+Install the [JavaScript Appwrite SDK](https://appwrite.io/docs/sdks).
+
+```sh
+npm install appwrite
+```
+
+Install the [Appwrite CSS library](https://pink.appwrite.io/) and icons package. This will give us a nice looking UI for our app. We just have to use the classes provided by the CSS library.
+
+```sh
+npm install "@appwrite.io/pink" "@appwrite.io/pink-icons"
+```
+
+Import the CSS file and icons in the `pages/_app.js` file.
+
+```js
+import "@appwrite.io/pink";
+import "@appwrite.io/pink-icons";
+```
+
+We will be using cookie to store the session token. So we need to install `next-cookie` package. We will also be using `react-hot-toast` to show toast messages.
+
+```bash
+npm install next-cookie react-hot-toast
+```
+
+We also have to add the `Toaster` component in the `pages/_app.js` file. This component will render all toasts.
```

Comment:
I don't have a pages directory that gets generated, where does this file come from?

Reply:
In the app creation time, we must check the `pages` directory to yes. I am using the Next.js `pages` router instead of the `app` router as it is beginner's friendly.

I can change the tutorial to the `app` directory if you want.